README.md

```diff
@@ -1,6 +1,6 @@
 <div align="center">
 
-[![Test Coverage](https://api.codeclimate.com/v1/badges/407cdb68165fc000cfd6/test_coverage)](https://codeclimate.com/github/CatalysmsServerManager/7-days-to-die-server-manager/test_coverage)
+[![Build Status](https://codecov.io/gh/CatalysmsServerManager/7-days-to-die-server-manager/branch/master/graph/badge.svg)](https://codecov.io/gh/CatalysmsServerManager/7-days-to-die-server-manager)
```

Comment:
This doesn't show 'Build status' but 'Test Coverage'

Reply:
I don't get it. Could you give me some more details...

Reply:
Changed that status.